### ISSUE TITLE
add HSTS headers in NGINX conf file

### ISSUE THREAD

**Issue by samcharles93** [SS88UK/VestaCP-Server-Installer#4]
Headers to be added to nginx conf file, to enable HSTS

add_header Strict-Transport-Security "max-age=31536000; includeSubdomains; preload";

**Comment by Neoboffin** [comment]
I agree, I have this set-up however, should they let the certificate expire, they can no longer browse the website because Chrome blocks access without a valid certificate. Not sure about other browsers.
